login.py

```
@@ -4,7 +4,7 @@
 def login(username, password):
```

Ruff (0.14.6)

Unused function argument: `username`

(ARG001)